estate/models/estate_property.py

```diff
@@ -0,0 +1,23 @@
+from odoo import models,fields
+
+class estateproperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Property"
+
+    name = fields.Char(required='True')
```

Comment:
It will throw an error. It takes a boolean value. Test your code before pushing it to GitHub.
```suggestion
    name = fields.Char(required=True)
```